packages/produce-promise/src/producePromise.js

```diff
@@ -33,8 +33,6 @@ export function producePromise() {
   /** @type {(reason: any) => void} */
   let rej = NOOP_INITIALIZER;
 
-  // We use a HandledPromise so that we can run HandledPromise.unwrap(p)
-  // even if p doesn't travel through a comms system (like SwingSet's).
   const p = new HandledPromise((resolve, reject) => {
```

Comment:
Even without that rationale, I still prefer `HandledPromise` and so left the code alone.